ingestion/src/metadata/utils/dbt_config.py

```diff
@@ -69,7 +70,7 @@ def _(config: DbtLocalConfig):
         return (
             json.loads(dbt_catalog),
             json.loads(dbt_manifest),
-            json.loads(dbt_run_results),
+            json.loads(dbt_run_results) if dbt_run_results else None,
```

Comment:
thanks for the change, that's what I meant in a previous comment hehe. Thanks for tackling it

Reply:
Yes I doubled checked it after that and found the issue...thanks for the comment 👍🏻